pkg/codegen/go/gen.go

```diff
@@ -276,8 +276,12 @@ func (pkg *pkgContext) inputType(t schema.Type, optional bool) string {
 	var typ string
 	switch t := t.(type) {
 	case *schema.EnumType:
+		var enumType string
```

Comment:
Nit: `enumType` doesn't feel like an obvious name. Is it more of a `modifier` thing, or `prefix`, or `nullable`?

Reply:
Fixed.